### DISCUSSION TITLE
Coder is unable to connect from coder application to azure windows machine

### DISCUSSION THREAD

**Discussion by DevOpsByMK** [coder/code-server#6146] [General]
Hi Team,

I have implemented templates for azure windows vm and azure linux vm.
After creating work space using azure linux vm template it was showing error message like "Main: linux unable to connect". Since my coder application is having trusted certificate it was not able to connect. So I have copied the certificate to the workspace manually and installed it. Right after that coder application was able to connect the linux work spcae. Below the screenshot for reference: 

![image](https://user-images.githubusercontent.com/98137876/231710831-b3d16df4-a8b8-4871-8de4-5e22c4bce340.png)

Same actions I have performed for azure windows workspace, still it shows the same error message like "Main: linux unable to connect".

![image](https://user-images.githubusercontent.com/98137876/231711180-b2712d14-6240-4ebc-b61b-e40f3336a077.png)

Please help me to resolve the issue.

Note: AWS windows vm is working properly.

**Comment by code-asher** [comment]
Just FYI I transferred your issue to the right repo: https://github.com/coder/coder/issues/7125.  This repo is for code-server rather than coder.  The naming can be a bit confusing!
